config/locales/en.yml

```diff
@@ -141,7 +141,7 @@ en:
       above_min: "%{field} must be at least %{min}"
     date:
       invalid_date: "Enter a date in the correct format, for example 31 1 2022"
-      outside_collection_window: Enter a date within the 22/23 financial year, which is between 1st April 2022 and 31st March 2023
+      outside_collection_window: Enter a date within the 22/23 collection year, which is between 1st April 2022 and 31st March 2023
```

Comment:
may be for next time i think it would be great it we could pass these values in

Reply:
this is only used for bulk upload, but yep I agree - probably a separate ticket